apps/www/components/LaunchWeek/lw7_days.ts

```diff
@@ -238,16 +277,219 @@ const days: WeekDayProps[] = [
   },
   {
     title: '',
-    shipped: false,
+    shipped: true,
     date: '14 Apr',
-    publishedAt: '2023-04-14T07:00:00.000-07:00',
+    publishedAt: '2023-04-13T00:00:00.000-07:00',
     description: '',
     d: 5,
     dd: 'Fri',
     youtube_id: '',
     blogpost: '',
     docs: '',
-    steps: [],
+    steps: [
+      {
+        title: 'Community Highlight',
+        hideInBlog: true,
+        steps: [
+          {
+            title: 'Community Highlight',
+            blog: '/blog/launch-week-7-community-highlights',
+            thumb: images['05-community-01-thumb'],
+            bg_layers: [
+              {
+                img: images['05-community-01-001'],
+              },
+              {
+                img: images['05-community-01-002'],
+              },
+            ],
+            steps: [],
+          },
+        ],
+      },
+      {
+        title: 'Studio Updates',
+        blog: '/blog/studio-v2',
+        thumb: images['05-studio-01-thumb'],
+        steps: [
+          {
+            title: '⌘K',
+            badge: 'New',
+            className: 'span-full',
+            url: '#command-k',
```

Comment:
@saltcod and we're linking to each blog post section from each card, can you check these?